iothub_client/tests/common_longhaul/iothub_client_common_longhaul.c

```diff
@@ -399,14 +401,48 @@ static char* create_twin_desired_properties_update(const char* test_id, unsigned
     return result;
 }
 
+static void validate_internet_conductivity()
```

Comment:
Your EE degree is pulling you back :)
```suggestion
static void validate_internet_connectivity()
```

Reply:
here and in the logs